File: Arnav Sharma/Assignment-0/if-else/discount.java
// A shop will give discount of 10% if the cost of purchased quantity is more than 1000.
// Ask user for quantity
// Suppose, one unit will cost 100.
// Judge and print total cost for user.
import java.util.Scanner;

public class discount {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter quantity : ");
        int q = sc.nextInt();

        if ((q * 100) > 1000) {
            System.out.println("discount "+(.1*q*100)+"total cost "+(q*100-(.1*q*100)));
        }
        else {
            System.out.println("no discount");
        }
        sc.close();
    }
}

File: Aishwarya Kadu/if else/discount.cpp
#include <iostream>
int main()
{
	using namespace std;
	int quantity,price;
	cout << "Enter quantity" << endl;
	cin >> quantity;
	price = quantity*100;
	if (price>1000){
		cout << "Total cost is " << price-(price*.1) << endl;
	}
	else{
		cout << "Total cost is " << price << endl;
	}
	return 0;
}